editoast/src/redis_utils.rs

```diff
@@ -1,28 +1,32 @@
 use crate::client::RedisConfig;
 use crate::error::Result;
+use futures::FutureExt;
 use redis::aio::{ConnectionLike, ConnectionManager};
 use redis::cluster::ClusterClient;
 use redis::cluster_async::ClusterConnection;
+use redis::AsyncCommands;
+use redis::Client;
+use redis::ErrorKind;
+use redis::Expiry;
+use redis::RedisError;
+use redis::RedisFuture;
 use redis::RedisResult;
-use redis::{AsyncCommands, RedisError};
-use redis::{Client, ToRedisArgs};
-use redis::{ErrorKind, Expiry};
+use redis::ToRedisArgs;
 use serde::de::DeserializeOwned;
 use serde::Serialize;
 
 pub enum RedisConnection {
     Cluster(ClusterConnection),
     Tokio(ConnectionManager),
+    Fake,
 }
 
 impl ConnectionLike for RedisConnection {
-    fn req_packed_command<'a>(
-        &'a mut self,
-        cmd: &'a redis::Cmd,
-    ) -> redis::RedisFuture<'a, redis::Value> {
+    fn req_packed_command<'a>(&'a mut self, cmd: &'a redis::Cmd) -> RedisFuture<'a, redis::Value> {
         match self {
             RedisConnection::Cluster(connection) => connection.req_packed_command(cmd),
             RedisConnection::Tokio(connection) => connection.req_packed_command(cmd),
+            RedisConnection::Fake => (async move { Ok(redis::Value::Nil) }).boxed(),
```

Comment:
You may use `futures::future::ok()` I believe. Not necessarily shorter, but looks a bit less like a shenanigan maybe🤷 

```suggestion
            RedisConnection::Fake => futures::future::ok(redis::Value::Nil).boxed(),
```